packages/types/src/simple-types.ts

```diff
@@ -1,10 +1,14 @@
-import { BigNumberish } from "ethers/utils";
+import { BigNumberish, Interface, ParamType } from "ethers/utils";
 
 export type ABIEncoding = string;
 export type AppInstanceID = string;
 export type Address = string;
 export type Bytes32 = string;
 
+// This is copied from the ethers definition of how an ABI is typed.
+// tslint:disable-next-line:prefer-array-literal
+export type ContractABI = Array<string | ParamType> | string | Interface;
```

Comment:
Does ethers not export it? Why do we need it 

Also why are you disabling the linter

Reply:
I thought it did but apparently not.

The linter doesn't like it when `Array` is used instead of an array literal (which honestly seems like a sub-optimal restriction - what's the disadvantage in using `Array`?)